### PR TITLE
developer's overview page updated

### PR DESCRIPTION
**Pull request by gulshanpr**
## Overview
PR for issue #963

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Comment by jcstein** [comment]
thanks for the PR @gulshanpr !

we recently added a FAQs page at the bottom of the category, if you'd like to add this, too

File: docs/developers/overview.md [hunk 1]
```diff
@@ -5,15 +5,29 @@ description: All things developer-related for building on top of Celestia.
...
+* How to [deploy rollups with](../../category/deploy-a-rollup/)
   * [Rollkit](../../developers/rollkit/)
   * [OP Stack](../../category/optimism/)
+  * [Sovereign SDK](https://github.com/Sovereign-Labs/sovereign-sdk/tree/stable/examples/demo-rollup#demo-rollup)
+  * [Dymension](https://dymension.xyz/)
+    * Full stack modular [blockchain develpment guide](../../developers/full-stack-modular-development-guide/)
+    * Setup [Relayer for IBC](../../developers/ibc-relayer/)
 * [Demos](../../developers/demos/) on Celestia
+* [Integrate with Celestia Network](../../developers/integrate-celestia/) for third-party service providers
+
```
Comment (jcstein) [inline docs/developers/overview.md:32]:
```suggestion
* [Frequently Asked Questions (FAQs)](https://docs.celestia.org/developers/faqs/)
```

File: docs/developers/overview.md [hunk 2]
```diff
@@ -5,15 +5,29 @@ description: All things developer-related for building on top of Celestia.
 
 # Developers
 
-This section contains all things developer-related for building on top
+This section contains all developer-related things for building on top
```
Comment (jcstein) [inline docs/developers/overview.md:8]:
```suggestion
This section contains all things developer-related for building on top
```

**Comment by gulshanpr** [comment]
@jcstein can you review the changes?

File: docs/developers/overview.md [hunk 3]
```diff
@@ -10,10 +10,25 @@ of Celestia.
 
 It includes:
 
-* An overview of [wallets on Celestia](../../developers/wallets/)
-* How to [interact with the Node API](../../developers/node-api/)
-* How to [deploy sovereign rollups](../../category/deploy-a-rollup/)
-* How to [integrate with Celestia](../../developers/integrate-celestia/)
+* Integration of [wallets on Celestia](../../developers/wallets/)
```
Comment (jcstein) [inline docs/developers/overview.md:13]:
```suggestion
* Overview of [wallets on Celestia](../../developers/wallets/)
```

File: docs/developers/overview.md [hunk 4]
```diff
@@ -10,10 +10,25 @@ of Celestia.
...
+  * [Keplr](../../developers/keplr/)
+  * [Cosmostation](../../developers/cosmostation/)
+  * Generate a wallet with
+    * [Celestia App](../../developers/celestia-app-wallet/)
+    * [Celestia Node](../../developers/celestia-node-key/)
+* How to [interact with Celestia Node API](../../developers/node-api/)
+  * how to use the [Celestia Node RPC API](../../developers/node-tutorial/)
```
Comment (jcstein) [inline docs/developers/overview.md:20]:
```suggestion
  * How to use the [Celestia Node RPC API](../../developers/node-tutorial/)
```

File: docs/developers/overview.md [hunk 5]
```diff
@@ -10,10 +10,25 @@ of Celestia.
...
-* How to [interact with the Node API](../../developers/node-api/)
-* How to [deploy sovereign rollups](../../category/deploy-a-rollup/)
-* How to [integrate with Celestia](../../developers/integrate-celestia/)
+* Integration of [wallets on Celestia](../../developers/wallets/)
+  * [Keplr](../../developers/keplr/)
+  * [Cosmostation](../../developers/cosmostation/)
+  * Generate a wallet with
+    * [Celestia App](../../developers/celestia-app-wallet/)
+    * [Celestia Node](../../developers/celestia-node-key/)
+* How to [interact with Celestia Node API](../../developers/node-api/)
```
Comment (jcstein) [inline docs/developers/overview.md:19]:
```suggestion
* How to [interact with the Celestia Node API](../../developers/node-api/)
```

File: docs/developers/overview.md [hunk 6]
```diff
@@ -10,10 +10,25 @@ of Celestia.
...
 
-* An overview of [wallets on Celestia](../../developers/wallets/)
-* How to [interact with the Node API](../../developers/node-api/)
-* How to [deploy sovereign rollups](../../category/deploy-a-rollup/)
-* How to [integrate with Celestia](../../developers/integrate-celestia/)
+* Integration of [wallets on Celestia](../../developers/wallets/)
+  * [Keplr](../../developers/keplr/)
+  * [Cosmostation](../../developers/cosmostation/)
+  * Generate a wallet with
+    * [Celestia App](../../developers/celestia-app-wallet/)
```
Comment (jcstein) [inline docs/developers/overview.md:17]:
```suggestion
    * [Celestia app](../../developers/celestia-app-wallet/)
```

File: docs/developers/overview.md [hunk 7]
```diff
@@ -10,10 +10,25 @@ of Celestia.
...
+    * [Celestia Node](../../developers/celestia-node-key/)
```
Comment (jcstein) [inline docs/developers/overview.md:18]:
```suggestion
    * [Celestia node](../../developers/celestia-node-key/)
```

File: docs/developers/overview.md [hunk 8]
```diff
@@ -10,10 +10,25 @@ of Celestia.
...
+* How to [interact with Celestia Node API](../../developers/node-api/)
+  * how to use the [Celestia Node RPC API](../../developers/node-tutorial/)
+  * Learn [Celestia Node API](../../developers/prompt-scavenger/) with a game
```
Comment (jcstein) [inline docs/developers/overview.md:21]:
```suggestion
  * Learn [Celestia node API](../../developers/prompt-scavenger/) with a game
```

File: docs/developers/overview.md [hunk 9]
```diff
@@ -10,10 +10,25 @@ of Celestia.
...
+  * [Node API docs](https://node-rpc-docs.celestia.org/)
+* How to [deploy rollups with](../../category/deploy-a-rollup/)
   * [Rollkit](../../developers/rollkit/)
   * [OP Stack](../../category/optimism/)
+  * [Sovereign SDK](https://github.com/Sovereign-Labs/sovereign-sdk/tree/stable/examples/demo-rollup#demo-rollup)
+  * [Dymension](https://dymension.xyz/)
+    * Full stack modular [blockchain develpment guide](../../developers/full-stack-modular-development-guide/)
```
Comment (jcstein) [inline docs/developers/overview.md:28]:
```suggestion
* Full stack modular [blockchain develpment guide](../../developers/full-stack-modular-development-guide/)
```

File: docs/developers/overview.md [hunk 10]
```diff
@@ -10,10 +10,25 @@ of Celestia.
...
+    * Setup [Relayer for IBC](../../developers/ibc-relayer/)
```
Comment (jcstein) [inline docs/developers/overview.md:29]:
```suggestion
```

File: docs/developers/overview.md [hunk 11]
```diff
@@ -10,10 +10,24 @@ of Celestia.
...
+* How to [deploy rollups with](../../category/deploy-a-rollup/)
   * [Rollkit](../../developers/rollkit/)
   * [OP Stack](../../category/optimism/)
+  * [Sovereign SDK](https://github.com/Sovereign-Labs/sovereign-sdk/tree/stable/examples/demo-rollup#demo-rollup)
+  * [Dymension](https://dymension.xyz/)
+* Full stack modular [blockchain develpment guide](../../developers/full-stack-modular-development-guide/)
 * [Demos](../../developers/demos/) on Celestia
+* [Integrate with Celestia Network](../../developers/integrate-celestia/) for third-party service providers
+* [Frequently Asked Questions (FAQs)](../../developers/faqs/)
+* [Frequently Asked Questions (FAQs)](https://docs.celestia.org/developers/faqs/)
```
Comment (jcstein) [inline docs/developers/overview.md:32]:
```suggestion
```

**Comment by gulshanpr** [comment]
@jcstein, I want to contribute in this repo, can you suggest me some beginner friendly issue that i can work on.

**Comment by jcstein** [comment]
https://github.com/celestiaorg/docs/issues?q=is:open+is:issue+label:%22good+first+issue%22

**Pull request merged**